specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2021-04-01-preview/keyvault.json

```diff
@@ -1320,7 +1325,11 @@
           "x-ms-enum": {
             "name": "CreateMode",
             "modelAsString": false
-          }
+          },
```

Comment:
Why was this added? It should have `read` as well (our generators will always construct a readable property), in which case it has all three possible values it would've had already, i.e. this seems redundant.

Reply:
It was write-only property in the previous version and is caused by swagger correctness issues before: https://github.com/Azure/azure-rest-api-specs/pull/13196/files/bf3d9933cef9afd516920b0a43ccf890f9c7533f#r584399451